test/operator/getters.go

```diff
@@ -0,0 +1,134 @@
+package operator
+
+import (
+	"context"
+
+	"github.com/maistra/istio-workspace/pkg/apis/istio/v1alpha1"
+
+	"github.com/onsi/gomega"
+	osappsv1 "github.com/openshift/api/apps/v1"
+	istionetwork "istio.io/client-go/pkg/apis/networking/v1alpha3"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+/*
+ * Test helpers for Operator test suite
+ */
+
+// New returns a new set of helpers for a given Client.
+func New(c client.Client) *Helpers {
+	return &Helpers{
+		Session:                   Session(c),
+		Gateway:                   Gateway(c),
+		DestinationRule:           DestinationRule(c),
+		VirtualService:            VirtualService(c),
+		VirtualServices:           VirtualServices(c),
+		Deployment:                Deployment(c),
+		DeploymentWithError:       DeploymentWithError(c),
+		DeploymentConfig:          DeploymentConfig(c),
+		DeploymentConfigWithError: DeploymentConfigWithError(c),
+	}
+}
+
+// Helpers simple struct to hold funcs.
+type Helpers struct {
```

Comment:
I think we should work a bit on the naming here. Is it operator, helper, getter, or bunch of builders?

Reply:
They are helpers for the test suite that makes it easier/cleaner to get objects from the client. They don't really build anything.

Reply:
https://github.com/maistra/istio-workspace/pull/469/commits/597ac6def9ade65dd8ef08a58a84741ada556acf